src/app/genes/gene-overview/gene-overview.component.ts

```diff
@@ -379,7 +379,11 @@ export class GeneOverviewComponent implements OnInit, OnDestroy, AfterContentChe
 
     removeNoDataTabs() {
         const itemsToCheck = [this.geneInfo.nominations, this.geneInfo.druggability, this.geneExpValidation];
-        const labels = [this.tabMenuLabels.NOMINATION_DETAILS, this.tabMenuLabels.NOMINATION_DETAILS, this.tabMenuLabels.EXPERIMENTAL_VALIDATION];
+        const labels = [
+            this.tabMenuLabels.NOMINATION_DETAILS,
+            this.tabMenuLabels.NOMINATION_DETAILS,
```

Comment:
Any idea why we have NOMINATION_DETAILS here twice? I wonder if the second one should correspond to druggability to match what's in itemsToCheck...

Reply:
Not sure, I never took a look at this. Might be worth creating a ticket to figure out if this may be causing a bug